app/models/admin_user.rb

```diff
@@ -25,8 +25,21 @@
 #
 
 class AdminUser < ApplicationRecord
+  include Ransackable
```

Comment:
Maybe we could just `include Ransackable` in `ApplicationRecord` so we don't need to remember to do it on every single model?